src/utils/address.ts

```diff
@@ -0,0 +1,17 @@
+import { ethers } from "ethers";
+
+export const ethereumAddressRegex = "^0x[a-fA-F0-9]{40}$";
+export const ensRegex = "[a-z]+.eth$";
+
+export function isValidAddress(address: string): boolean {
+  if (typeof address !== "string") {
+    return false;
+  }
+  return address.match(ensRegex)
+    ? ethers.utils.isValidName(address)
+    : ethers.utils.isAddress(address);
+}
+
+export function splitAddress(address: string) {
+  return `${address.slice(0, 8)}...${address.slice(-6)}`;
```

Comment:
not sure i understand what this is for?

Reply:
This is just to shorten an address and display it in the UI, with 3 dots in the middle